packages/babel-plugin-transform-es2015-shorthand-properties/README.md

```diff
@@ -30,7 +30,7 @@ var cat = {
 
 ```js
 var cat = {
-  getName: function () {
+  getName: function getName() {
```

Comment:
When you run Try on the [plugin's page on the website](https://babeljs.io/docs/plugins/transform-es2015-shorthand-properties/). The preset es2015 is enable by default.